api/src/main/java/com/codeforcommunity/rest/subrouter/authenticated/ProtectedSchoolRouter.java

```diff
@@ -346,4 +358,29 @@ private void handleUpdateBookLog(RoutingContext ctx) {
     processor.updateBookLog(userData, schoolId, bookId, request);
     end(ctx.response(), 200);
   }
+
+  private void handleGetWithoutLibraryReportAsCsv(RoutingContext ctx) {
+    JWTData userData = ctx.get("jwt_data");
+    int reportId = RestFunctions.getPathParamAsInt(ctx, "report_id");
+    String response;
+    try {
+      response = processor.getReportAsCsv(userData, reportId, false);
```

Comment:
(See comment for `ProtectedSchoolProcessorImpl`)

Let's remove the `try` / `catch` from here. If something goes wrong, we should throw a custom exception and return an error to the user, rather than returning a `200` and an empty string